### DISCUSSION TITLE
Concourse API Plan

### DISCUSSION THREAD

**Discussion by ghost** [concourse/concourse#5406] [General]
thanks. a login api should be considerable.
"/api/v1/login" (edited) 

Ye Yang  13 hours ago
for "/api/v1/builds/:build_id/events"  it seems now work in http streaming mode (server sent event: https://en.wikipedia.org/wiki/Server-sent_events). so an api for build logs would be highly appreciated.
"/api/v1/builds/:build_id/build_log" (edited) 

Ye Yang  13 hours ago
the api for build log should provide:
queries on aged builds
metadata from resources should be included in build logs
pagination for better sql performance if necessary
in structured formats (json, xml)
downloadable (txt, or zip)

**Comment by vito** [comment]
?
